src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
+
+  return (
+    <a className="UserInfo" href={email}>
```

Comment:
Please fix the link
![image](https://user-images.githubusercontent.com/98764641/234226206-b9eb5053-3b54-4458-b55e-af29aff19e2f.png)